### PR TITLE
3.- Game

### PR DESCRIPTION
**Pull request by pfariaz**
# Summary
-new endpoint for create a game (POST)
-new endpoint for list existing games (GET)
-new endpoint for create a match game (POST)
-new endpoint for list existing match games (GET)
-and test for each new endpoint

### CODE REVIEW

File: test/games.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
```
Comment (hdf1996) [inline test/games.js]: When you do `../` you don't need to do `./` again

File: test/games.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            res.should.have.status(200);
+            dictum.chai(res);
+          })
+          .then(() => done());
+      });
+    });
+  });
+
+  describe('/games/:game_id/match GET', () => {
+    it('should be success', done => {
```
Comment (hdf1996) [inline test/games.js:127]: What about the bad case?

File: test/games.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            res.should.be.json;
+            dictum.chai(res);
+          })
+          .then(() => done());
+      });
+    });
+  });
+
+  describe('/games/:game_id/match POST', () => {
+    it('should be success', done => {
```
Comment (hdf1996) [inline test/games.js:70]: What about the bad case?

File: test/games.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            dictum.chai(res);
+          })
+          .then(() => done());
+      });
+    });
+  });
+
+  describe('/games GET', () => {
+    it('should be success', done => {
+      login((loginError, loginResponse) => {
```
Comment (hdf1996) [inline test/games.js]: What about the bad case?

File: test/games.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    .end((error, respose) => {
+      if (callback) {
+        callback(error, respose);
+      }
+    });
+};
+
+describe('games', () => {
+  describe('/games POST', () => {
+    it('should be success', done => {
```
Comment (hdf1996) [inline test/games.js]: What about the bad case?

File: test/users.js [hunk 6]
```diff
@@ -0,0 +1,195 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  sessionManager = require('../app/services/sessionManager'),
+  should = chai.should();
+
+const successfulLogin = callback => {
```
Comment (hdf1996) [inline test/users.js]: Can't you reuse this function? it seems the same than the game's login function

**Review by hdf1996 — CHANGES_REQUESTED** [review]

File: test/users.js (hunk 6 above)
Comment (pfariaz) [reply test/users.js]: now i added a new script js called utils.js with the login function

File: test/games.js (hunk 2 above)
Comment (pfariaz) [reply test/games.js:127]: now i added all bad cases for all games endpoints

**Review by hdf1996 — APPROVED** [review]

**Pull request merged**